spec.md

```diff
@@ -614,6 +614,12 @@ This RPC will be called by the CO to provision a new volume on behalf of a user
 This operation MUST be idempotent.
 If a volume corresponding to the specified volume `name` already exists, is accessible from `accessibility_requirements`, and is compatible with the specified `capacity_range`, `volume_capabilities` and `parameters` in the `CreateVolumeRequest`, the Plugin MUST reply `0 OK` with the corresponding `CreateVolumeResponse`.
 
+Plugins MAY create 3 types of volumes:
+
+- Empty volumes. When plugin supports `CREATE_DELETE_VOLUME` optional capability.
+- From an existing snapshot. When plugin supports `CREATE_DELETE_SNAPSHOT` optional capability.
```

Comment:
When plugin supports `CREATE_DELETE_VOLUME` and `CREATE_DELETE_SNAPSHOT` optional capabilities.